### DISCUSSION TITLE
Switch from Database to Local file system and vice versa

### DISCUSSION THREAD

**Discussion by steam3d** [usememos/memos#2548] [Q&A] [answered]
Could I switch between Database and Local file system during usage?  (For example, I used the Memos couple months and decided that local file system is better for me). 

I saw you have radio button, but will it migrate my data?
![image](https://github.com/usememos/memos/assets/30986874/c5c493f9-351a-4c25-b1ce-b1c0b3019301)

I also checked this: https://github.com/orgs/usememos/discussions/1948. If I understood right, text always stored in database, but what the local storage does then? Just stores media files without notes?

**Comment by MichaKersloot** [comment, marked as answer]
Just did exactly that. It didn't migrate the assets (like images) from database to the filesystem, but new uploaded files will be stored in the filesystem from now on. This while the old assets that are already in the database are still available in the memo's.
